rasa_nlu/evaluate.py

```diff
@@ -122,18 +122,23 @@ def plot_confusion_matrix(cm, classes,
         fig.savefig(out, bbox_inches='tight')
 
 
-def plot_histogram(hist_data,
-                   out=None):  # pragma: no cover
-    """Plot a histogram of the confidence distribution of the predictions.
-
+def plot_histogram(hist_data, out=None):    # pragma: no cover
+    """Plot a histogram of the confidence distribution of the predictions in
+    two columns.
+    Green for the confidences of hits, Red for the confidences of misses.
```

Comment:
Please use two colors that can be read by people with deuteranopia.

i'd suggest
- positive: #009292 (wine-ish)
- negative: #920000 (blueish)

Reply:
Very good point!
Ofc, the change is in its way